mariadb/content.md

```diff
@@ -157,3 +157,11 @@ Most of the normal tools will work, although their usage might be a little convo
 ```console
 $ docker exec some-%%REPO%% sh -c 'exec mysqldump --all-databases -uroot -p"$MYSQL_ROOT_PASSWORD"' > /some/path/on/your/host/all-databases.sql
 ```
+
+## Restoring data from dump files
+
+For restoring data. You can print the dump file as standard output, and directly pipe it to the `docker exec` command with `-i` flag:
+
+```console
+$ cat /some/path/on/your/host/all-databases.sql | docker exec -i some-%%REPO%% sh -c 'exec mysql -uroot -p"$MYSQL_ROOT_PASSWORD"'
```

Comment:
Can we adjust this to instead use `<` both for slightly better resource usage (piping direct from the file/Bash to `mysql` inside Docker instead of involving `cat` as well) and for better symmetry with the backup example above?  ala:

```console
$ docker exec -i some-%%REPO%% sh -c 'exec mysql -uroot -p"$MYSQL_ROOT_PASSWORD"' < /some/path/on/your/host/all-databases.sql
```

Reply:
@tianon Fair point 😃 
I will adjust the command and the description.